doc/source/notebooks/manual/advanced/settings.ipynb

```diff
@@ -4,11 +4,228 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Settings and GPflow config\n",
-    "--\n",
+    "# Settings and GPflow config\n",
+    "GPflow has a config file, `gpflowrc` which allows the user to change the default behavious in GPflow. GPflow searches for the file in the following order:\n",
+    "1. In the working directory\n",
+    "2. In the user's home directory\n",
+    "3. In the GPflow directory (revert to default)\n",
     "\n",
-    "see settings.ipynb"
+    "You can also make `gpflowrc` a hidden file, if you don't want it clutting your home directory, by renaming as `.gpflowrc`.\n",
+    "\n",
+    "Usually, one wants to change the settings for an entire project. We recommend adding a `gpflowrc` file in the working directory to achieve this effect. It is helpful to add assertions / warnings in code to ensure that the correct settings are used. GPflow also allows settings to be adjusted in code, although this comes with some additional difficulties (see below). This is why we recommend using a settings file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Common uses\n",
```

Comment:
Maybe an extra 'common use' is changing the logging level. Really useful to see which multiple dispatch methods are actually being used.

Reply:
This is now addressed. However, I couldn't get it to work by changing the logging level in code. It only seemed to work if I changed it in `gpflowrc`.